### PR TITLE
Use the latest available ASM version

### PR DESCRIPTION
**Pull request by keithc-ca**
The plugin manager will fail to load all the expected classes if it encounters any that use class-file features not supported by the specified ASM version.

Fixes: #9985.

### CODE REVIEW

File: jcl/src/openj9.dtfj/share/classes/com/ibm/java/diagnostics/utils/plugins/impl/ClassScanner.java [hunk 1]
```diff
@@ -36,26 +36,39 @@
...
 	private ClassInfo info;
 	private Annotation currentAnnotation = null;
-	private final URL url;			//where the class is being scanned from
+	private final URL url; // where the class is being scanned from
 	private final Set<ClassListener> listeners;
-	
+
 	public ClassScanner(URL url, Set<ClassListener> listeners) {
-		super(Opcodes.ASM4, null);
+		/*[IF Java15]*/
```
Comment (pshipton) [inline jcl/src/openj9.dtfj/share/classes/com/ibm/java/diagnostics/utils/plugins/impl/ClassScanner.java:47]: Seems we need a persistent issue to keep this updated. Java 16 builds will start happening soon I hope.

**Comment by pshipton** [comment]
jenkins test extended plinux jdk14

**Comment by pshipton** [comment]
jenkins compile plinux jdk8,jdk11

**Comment by pshipton** [comment]
Pls create a PR for the 0.21 branch, might as well fix the release.

File: jcl/src/openj9.dtfj/share/classes/com/ibm/java/diagnostics/utils/plugins/impl/ClassScanner.java (hunk 1 above)
Comment (keithc-ca) [reply jcl/src/openj9.dtfj/share/classes/com/ibm/java/diagnostics/utils/plugins/impl/ClassScanner.java:47]: Java 16 doesn't introduce any new ASM versions as compared with Java 15 right now, but, yes, that might change: I created #9990.

**Comment by keithc-ca** [comment]
> Pls create a PR for the 0.21 branch, might as well fix the release.

See #9991.

**Pull request merged**